e2e/cypress/integration/integrations/builtin_commands/common_commands_spec.js

```diff
@@ -72,6 +72,29 @@ describe('I18456 Built-in slash commands: common', () => {
             cy.get(`#postMessageText_${postId}`).should('have.text', 'test ¯\\_(ツ)_/¯');
         });
     });
+
+    it('MM-T710 /mute error message', () => {
+        loginAndVisitDefaultChannel(user1, testChannelUrl);
+
+        const invalidChannel = 'oppagangnamstyle';
+
+        // # Type /mute with random characters
+        cy.postMessage(`/mute ${invalidChannel}`);
+        cy.wait(TIMEOUTS.THREE_SEC);
```

Comment:
Please use `cy.uiWaitUntilMessagePostedIncludes` instead of explicit wait.
```suggestion
        cy.uiWaitUntilMessagePostedIncludes('Please use the channel handle to identify channels');
```